doc-enhancement/update PATH variable

angad-singhh opened the pull request:
- This PR improves the documentation regarding update PATH variable section.
- The PR fixes the issue #7344

ntrogh commented on docs/cpp/config-mingw.md:
```diff
@@ -51,12 +51,13 @@ To install the MinGW-w64 toolchain, check out this video or follow the steps bel
    1. In your **User variables**, select the `Path` variable and then select **Edit**.
    1. Select **New** and add the MinGW-w64 destination folder you recorded during the installation process to the list. If you used the default settings above, then this will be the path: `C:\msys64\ucrt64\bin`.
-   1. Select **OK** to save the updated PATH. You will need to reopen any console windows for the new PATH location to be available.
+   1. Once you have added the PATH in **Edit Environment Variable** window, select **OK**, and ensure to select **OK** again in the **Environment Variable** window to update the PATH.
```
```suggestion
   1. Select **OK**, and then select **OK** again in the **Environment Variables** window to update the `PATH` environment variable.
```

ntrogh commented on docs/cpp/config-mingw.md:
```diff
@@ -51,12 +51,13 @@ To install the MinGW-w64 toolchain, check out this video or follow the steps bel
    1. Select **New** and add the MinGW-w64 destination folder you recorded during the installation process to the list. If you used the default settings above, then this will be the path: `C:\msys64\ucrt64\bin`.
-   1. Select **OK** to save the updated PATH. You will need to reopen any console windows for the new PATH location to be available.
+   1. Once you have added the PATH in **Edit Environment Variable** window, select **OK**, and ensure to select **OK** again in the **Environment Variable** window to update the PATH.
+   1. You will have to open a NEW console window for the updated PATH to work.
```
```suggestion
      You have to reopen any console windows for the updated `PATH` environment variable to be available.
```

ntrogh commented on docs/languages/cpp.md:
```diff
@@ -92,12 +92,13 @@ To understand the process, let's install Mingw-w64 via [MSYS2](https://www.msys2
    1. In your **User variables**, select the `Path` variable and then select **Edit**.
    1. Select **New** and add the MinGW-w64 destination folder you recorded during the installation process to the list. If you selected the default installation steps, the path is: `C:\msys64\ucrt64\bin`.
-   1. Select **OK** to save the updated PATH. For the new `PATH` to be available, reopen your console windows.
+   1. Once you have added the PATH in **Edit Environment Variable** window, select **OK**, and ensure to select **OK** again in the **Environment Variable** window to update the PATH.
```
```suggestion
   1. Select **OK**, and then select **OK** again in the **Environment Variables** window to update the `PATH` environment variable.
```

ntrogh commented on docs/languages/cpp.md:
```diff
@@ -92,12 +92,13 @@ To understand the process, let's install Mingw-w64 via [MSYS2](https://www.msys2
    1. Select **New** and add the MinGW-w64 destination folder you recorded during the installation process to the list. If you selected the default installation steps, the path is: `C:\msys64\ucrt64\bin`.
-   1. Select **OK** to save the updated PATH. For the new `PATH` to be available, reopen your console windows.
+   1. Once you have added the PATH in **Edit Environment Variable** window, select **OK**, and ensure to select **OK** again in the **Environment Variable** window to update the PATH.
+   1. You will have to open a NEW console window for the updated PATH to work.
```
```suggestion
      You have to reopen any console windows for the updated `PATH` environment variable to be available.
```

ntrogh reviewed (changes requested):
@angad-singhh thanks for creating this PR. I've updated the wording slightly. I've changed the "reopening of console windows" from a bulleted item to just a note.

angad-singhh commented:
> @angad-singhh thanks for creating this PR. I've updated the wording slightly. I've changed the "reopening of console windows" from a bulleted item to just a note.

Thanks, for the suggestions. I have made commits for the same.

ntrogh reviewed (approved):
Thank you!

angad-singhh commented:
> @angad-singhh please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.

@microsoft-github-policy-service agree
